### PR TITLE
fix: deduplicate inherited members

### PR DESCRIPTION
**Pull request by sdkrystian**

### CODE REVIEW

**Comment by sdkrystian** [comment]
closes #721

File: src/lib/Metadata/Interface.cpp [hunk 1]
```diff
@@ -75,7 +75,8 @@ class TrancheBuilder
         std::vector<SymbolID>& M,
         const Info& I)
     {
-        M.emplace_back(I.id);
+        if(std::ranges::find(M, I.id) == M.end())
```
Comment (alandefreitas) [inline src/lib/Metadata/Interface.cpp:78]: Will this ensure we take the right inherited member, though?

Comment (sdkrystian) [reply src/lib/Metadata/Interface.cpp:78]: @alandefreitas All this does is ensure that elements in the tranche are unique.

Comment (alandefreitas) [reply src/lib/Metadata/Interface.cpp:78]: So let's at least open another issue

**Pull request merged**